tdex/umbrel-app.yml

```diff
@@ -0,0 +1,28 @@
+manifestVersion: 1.1
+id: tdex
+category: Finance
+name: TDEX Provider
+version: "0.8.15"
+tagline: Provide liquidity on the TDEX Network, an open protocol to trade Liquid Network assets.
+description: >-
+  A Provider holds Liquid reserves of both a BASE_ASSET-QUOTE_ASSET in his non-custodial Liquid hot-wallet, running automated market-making strategies, either with or without an oracle. Providers are incentivized to be always on and need to expose a public reachable endpoint either via clearnet or using a Onion hidden service.
+  A small provider's fee can be taken out of each trade and added to the reserves. 
+  A liquidity provider has full control over the market making strategy to apply needed to calculate the market rate at which to accept trades. That being said, there is a possibility to apply an automated market-making relying only on the reserves balances and the amount requested by the trader, without the need to connect to an external price feed. The default strategy of the alpha daemond is the constant product market-making. In short, this model generates a full order-book based on an initial price for the market. Every transaction that occurs on this market will adjust the prices of the market accordingly.
+releaseNotes: >-
```

Comment:
We can remove the entire releaseNotes section for the initial app release. This gets used once we start doing app updates.